ceph/ceph.py

```diff
@@ -2255,7 +2255,7 @@ def create_ceph_object(self, role):
             if len(free_volume_list) > 0:
                 free_volume_list[0].status = NodeVolume.ALLOCATED
             else:
-                raise RuntimeError("Insufficient of free volumes")
+                raise RuntimeError("Node/s with osd role do not have number of volumes specified")
```

Comment:
can  we specify expected number of volumes?

Reply:
The error was about the key no-of-volumes missing in the conf file. I have changed the error message now. 

Should we specify the number of volumes expected as well?

Reply:
Got it wrong by looking at PR title. its `no-of-volumes`.